pkg/monitor/monitor/util.go

```diff
@@ -139,6 +142,58 @@ func getMonitorConfigMap(tc *v1alpha1.TidbCluster, dc *v1alpha1.DMCluster, monit
 		EnableTLSDMCluster: dc != nil && dc.IsTLSClusterEnabled(),
 	}
 
+	if monitor.Spec.RemoteWrite != nil && len(monitor.Spec.RemoteWrite) > 0 {
+		var remoteWriteConfigs []*config.RemoteWriteConfig
+		for _, remoteWrite := range monitor.Spec.RemoteWrite {
+			url, err := client.ParseHostURL(remoteWrite.URL)
+			if err != nil {
+				continue
+			}
+			httpClientConfig := config.HTTPClientConfig{
+				BearerTokenFile: remoteWrite.BearerTokenFile,
+				TLSConfig: config.TLSConfig{
+					CAFile:             remoteWrite.TLSConfig.CAFile,
+					CertFile:           remoteWrite.TLSConfig.CertFile,
+					KeyFile:            remoteWrite.TLSConfig.KeyFile,
+					ServerName:         remoteWrite.TLSConfig.ServerName,
+					InsecureSkipVerify: remoteWrite.TLSConfig.InsecureSkipVerify,
+				},
+			}
+			var writeRelabelConfigs []*config.RelabelConfig
+			for _, writeRelabelConfig := range remoteWrite.WriteRelabelConfigs {
+				regex, err := config.NewRegexp(writeRelabelConfig.Regex)
+				if err != nil {
+					continue
+				}
+				writeRelabelConfigs = append(writeRelabelConfigs, &config.RelabelConfig{
+					SourceLabels: writeRelabelConfig.SourceLabels,
+					Separator:    writeRelabelConfig.Separator,
+					Regex:        regex,
+					Modulus:      writeRelabelConfig.Modulus,
+					TargetLabel:  writeRelabelConfig.TargetLabel,
+					Replacement:  writeRelabelConfig.Replacement,
+					Action:       writeRelabelConfig.Action,
+				})
+			}
+			remoteWriteConfigs = append(remoteWriteConfigs, &config.RemoteWriteConfig{
+				URL:                 &config.URL{URL: url},
+				RemoteTimeout:       remoteWrite.RemoteTimeout,
+				WriteRelabelConfigs: writeRelabelConfigs,
+				HTTPClientConfig:    httpClientConfig,
+				QueueConfig: config.QueueConfig{
+					Capacity:          remoteWrite.QueueConfig.Capacity,
```

Comment:
If not specified queue config, Capacity、MaxShards... will set to 0?

Reply:
![image](https://user-images.githubusercontent.com/13257127/104411563-92746b00-55a5-11eb-876c-93888979b8a2.png)

Reply:
We should only assign `Capacity、MaxShards...` value when QueueConfig is not nil，currently `remoteWrite.QueueConfig.Capacity` values are 0?

Reply:
![image](https://user-images.githubusercontent.com/13257127/104618101-43265b80-56c7-11eb-9acc-78e73b5ceee0.png)

Reply:
I think if we not set  `remoteWrite.QueueConfig.Capacity` field , it will use default value.